packages/vscode-extension/src/treeview/deploymentView.tsx

```diff
@@ -80,6 +80,14 @@ export class DeploymentView extends React.Component<any, any> {
           disable={this.state.locked}
           command="fx-extension.publish"
         ></TreeItem>
+        <TreeItem
+          label="Add CI/CD Workflows"
+          tooltip={StringResources.vsc.commandsTreeViewProvider.addCICDWorkflowsDescription}
```

Comment:
the content of tooltip and label are same, which is not meaningful. Tooltip need to explain more detail.

Reply:
Let me update it with PM, thanks.